### PR TITLE
Update novice

### PR DESCRIPTION
**Pull request by Tjoosten**

### CODE REVIEW

**Comment by kladd** [comment]
I know I used the '\n' line ending but while you're changing it would you mind instead doing `echo 'usage string' . PHP_EOL;`? Just in-case the platform doesn't use line feed endings? Thanks!

**Comment by Tjoosten** [comment]
Can u explain more?

File: novice [hunk 1]
```diff
@@ -18,7 +18,14 @@ class Novice {
...
     {
-        echo "usage: php " . $this->args[0] . " <command> [<args>]\n";
+        echo "\n";
+        echo "syntaxis: php novice <command> [<args>] \n\n";
+
+        echo "Commands: \n";
+        echo "php novice --help                  -->   Displays the help menu. \n";
+        echo "php novice migrate                 -->   Migrate the database. \n";
+        echo "php novice seed                    -->   Seed the database tables. \n";
+        echo "php novice novice migrate --seed   -->   Migrate and seed the database. \n\n";
```
Comment (kladd) [inline novice]: `novice` is here twice.

**Comment by kladd** [comment]
Instead of
```php
        echo "\n";
```
i was suggesting
```php
        echo PHP_EOL;
```

because it's platform independent.

**Comment by Tjoosten** [comment]
Alright i will work that out tomorrow!

**Comment by Tjoosten** [comment]
@kladd I changed ´\n´ to ´PHP_EOL´

**Pull request merged**